dp-game-webapp/app/about/page.tsx

```diff
@@ -0,0 +1,457 @@
+      <div className="mt-10">
+        <p className="mt-2">
+          This is a game designed to better understand the effect of
+          differential privacy on ads measurement. The{" "}
+          <a href="https://www.w3.org/community/patcg/" className="underline">
+            <i>Private Ads Technology Community Group</i> (PATCG)
+          </a>{" "}
+          in the World Wide Web Consortium (W3C) is actively working on a new
+          web standard for <i>private ads measurement</i> that can enable
+          aggregate measurement of ad effectiveness based on cross site activity
+          while preventing user tracking.
+        </p>
+        <p className="mt-2">
+          There is initial consensus in that group that{" "}
+          <a
+            href="https://github.com/patcg/docs-and-reports/blob/main/design-dimensions/Dimensions-with-General-Agreement.md#privacy-defined-at-least-by-differential-privacy"
+            className="underline"
+          >
+            differential privacy
+          </a>{" "}
+          will be part of the solution, and this effort attempts to better
+          understand the effect of that noise on the utility of ads measurement.
+        </p>
+
+        <h1 className="my-2 text-lg sm:text-4xl font-bold tracking-tight">
+          Differential Privacy
+        </h1>
+        <p className="mt-2">
+          We often think of privacy as a <i>binary</i> attribute: either a
+          system is private or it is not. Contrary to this idea, differential
+          privacy does not tell us if a system is private or not.
+        </p>
+        <p className="mt-2">
+          Instead, it's <b>measure of a system</b> that tells us how much
+          information is revealed about individual data points within that
+          system.
+        </p>
+        <p className="mt-2">
+          If you interested in the mathematical details,{" "}
+          <a
+            href="https://en.wikipedia.org/wiki/Differential_privacy#Definition_of_%CE%B5-differential_privacy"
+            className="underline"
+          >
+            check out Wikipedia
+          </a>
+          . Here, we'll instead motivate with an example.
+        </p>
+
+        <h1 className="my-2 text-lg sm:text-4xl font-bold tracking-tight">
+          Motivating Example
+        </h1>
+
+        <p className="mt-2">
+          Suppose we had access to a <b>database</b> that stores information
+          potentially sensitive data about people, such as if they have a
+          certain health condition. The database is operated by some trusted
+          source such as a health care provider, and it only allows{" "}
+          <b>aggregate</b> queries, say for the purpose of tracking prevalence
+          among the general population.
+        </p>
+        <p className="mt-2">
+          Now, let's say that <b>aggregation requirement</b> enforced by the
+          database results in an error if includes left than 100 people in the
```

Comment:
```suggestion
          database results in an error if includes less than 100 people in the
```

Reply:
good catch! making one small adjustment to the suggestion:
```suggestion
          database results in an error if it includes less than 100 people in the
```